exodus/restful_api/views.py

```diff
@@ -31,10 +31,17 @@ def get_report_infos(request, r_id):
         except Report.DoesNotExist:
             raise Http404('No report found')
 
+        certificate = None
+        if hasattr(report.application, 'apk'):
+            certificates = Certificate.objects.filter(id=report.application.apk.id)
+            if certificates.count() == 1:
```

Comment:
Could we simply use `if certificates:` here?

Reply:
@simpnu Thanks for your feedback ; I've added requested tests

Reply:
> Could we simply use `if certificates:` here?

done too